docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc

```diff
@@ -0,0 +1,83 @@
+
+= Cortex Time Series Plugin
+
+The Time Series Storage Plugin for Cortex enables {page-component-title} to persist aggregated flow data to Cortex.
+The plugin uses the xref:deployment:time-series-storage/timeseries/ts-integration-layer.adoc[time series storage integration layer] to convert metrics to a Prometheus model and delegate writes and reads to https://cortexmetrics.io/[Cortex].
+The plugin also makes it possible for users to visualize graphs from the Cortex instance in https://docs.opennms.com/helm/latest/index.html[OpenNMS Helm], through the OpenNMS performance data source or the https://grafana.com/grafana/plugins/prometheus/[Prometheus Data Source].
+
+.Time Series Storage Plugin for Cortex architecture
+image::time-series-storage/cortex-plugin.png[Cortex plugin architecture,500]
+
+== Requirements
+
+Make sure you have the following before you start to use the Cortex plugin:
+
+* Horizon 30.0.0+ (Meridian availability with the 2023 release)
+* Cortex version 1.10.0+
+
+NOTE: We plan to keep the plugin compatible with https://grafana.com/oss/mimir/[Mimir], Grafana's fork of Cortex, as long as Mimir remains compatible with Cortex.
+The minimum supported Mimir version is 2.0.0.
+
+== Deployment
+
+To use the Time Series Storage Plugin for Cortex, do the following:
+
+. Start Cortex.
++
+See https://cortexmetrics.io/docs/getting-started/.
++
+You can also download a https://github.com/opennms-forge/stack-play/tree/master/standalone-cortex-minimal[minimal, standalone version of Cortex] and start Cortex with the `docker-compose up` command.
```

Comment:
Let's rewrite to:
If your aim is to get up to speed quickly in a test environment, you can use Docker Compose to deploy a https://github.com/opennms-forge/stack-play/tree/master/standalone-cortex-minimal[minimal, standalone Cortex instance].